frontends/analytics/src/components/Sidebar.tsx

```diff
@@ -50,15 +50,38 @@ const navbarRoutes = [
   },
 ];
 
+const dashboardURL = import.meta.env.VITE_DASHBOARD_URL as string;
+
+const domainNavbarRoutes = [
+  {
+    href: dashboardURL,
+    label: "Dashboard",
+    icon: TbLayoutDashboard,
+  },
+  {
+    href: "https://docs.trieve.ai/api-reference/",
+    label: "API Docs",
+    icon: AiOutlineApi,
+  },
+  {
+    href: "https://search.trieve.ai",
+    label: "Search Playground",
+    icon: HiOutlineMagnifyingGlass,
+  },
+  {
+    href: "https://chat.trieve.ai",
+    label: "Chat Playground",
+    icon: IoChatboxOutline,
```

Comment:
These values should be env's and also link out to the playgrounds on the proper org and dataset similar to how dashboard links work. It's really disorienting when the link takes you to the wrong dataset.